ocf_ml_metrics/metrics/errors.py

```diff
@@ -187,24 +197,24 @@ def compute_metrics(
     start_time: np.ndarray,
     filter_by_night: bool = False,
     tag: str = "",
-    thresholds: Optional[Union[list, float]] = -1,
```

Comment:
`mypy` complained that `thresholds` must never be `None`. (And `Optional` is the same as `Union[..., None]`).